src/StudyEntities/Candidate/CandID.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace LORIS\StudyEntities\Candidate;
+class CandID extends ValidatableIdentifier
+{
+    public function getType(): string
+    {
+        return 'CandID';
+    }
+
+    static protected function validate(string $value): bool
```

Comment:
Comment here about what does/doesn't get validated (to address confusion like @paiva's question)